js/src/util/index.js

```diff
@@ -264,9 +264,9 @@ const execute = callback => {
 const getNextActiveElement = (list, activeElement, shouldGetNext, isCycleAllowed) => {
   let index = list.indexOf(activeElement)
 
-  // if the element does not exist in the list initialize it as the first element
+  // if the element does not exist in the list return an element depending on the direction and if cycle is allowed
   if (index === -1) {
-    return list[0]
+    return list[!shouldGetNext && isCycleAllowed ? list.length - 1 : 0]
```

Comment:
Talked with @GeoSot about adding the feature of getting the last item from `getNextActiveElement()` in case the item is not in the list, we want the previous item and cycling is allowed. (To reflect the case when the dropdown button is the target of the event and the `ARROW_UP_KEY` is pressed.) He was ok with adding it.

But I'd appreciate to receive some other opinion.